docs/source/index.rst

```diff
@@ -1,15 +1,20 @@
 ivadomed
 ========
```

Comment:
Yes the issue is coming from the title, but see comment [here](https://github.com/ivadomed/ivadomed/pull/419#pullrequestreview-475094131). .. :-(

Reply:
We need a title or else in the breadcrumb, it is shown that Contributors is the title of the page
![image](https://user-images.githubusercontent.com/49137243/91526066-d144a580-e8d0-11ea-9595-d8ec5fa57761.png)

But we could change ivadomed for something more general like Home Page or Summary

Reply:
hum... we should be able to omit the "home page" or "ivadomed" at the top, see eg [here](https://niftytorch.github.io/doc/)

Reply:
^ ah, nevermind, they are using MkDocs, so there might be some subtle differences...

Reply:
i'm definitely not a big fan of the "home page", so let's try to find a workaround...

Reply:
doing some tests locally-- a solution would be to move the contributors/license in separate files and be displayed in the TOC

Reply:
oops... when doing that, that's what we get 😞 

![image](https://user-images.githubusercontent.com/2482071/91560902-a118f900-e908-11ea-91c3-9a6d71ff3cbc.png)

Reply:
https://github.com/ivadomed/ivadomed/pull/419/commits/483acd51f993361cf219a23cd0e7d773ad01ba52 seems like a good compromise:

![image](https://user-images.githubusercontent.com/2482071/91561984-6f089680-e90a-11ea-92e7-fd1359f6efa1.png)

but now we're back to the list not displaying 😭

Reply:
ah! i have an idea: let's move the purpose to another section

Reply:
ok how about https://github.com/ivadomed/ivadomed/pull/419/commits/ad93601c4126fb73426270188129443e23358c35 ?

i actually find it better than previously-- it will allow us to also add more items in the list of purpose (two items was a bit sad)

Reply:
I also prefer to move the main heading from "comparison to other projects" to "purpose" :-)